runhouse/rns/secrets.py

```diff
@@ -385,10 +434,12 @@ def save_secrets(cls, secrets: dict, file_path: Optional[str] = None):
             value=secrets["secret_key"],
         )
 
-        cls.save_to_config_file(parser, dest_path)
+        if overwrite:
```

Comment:
It looks like we only write down the secrets if overwrite is True (even if there's no conflict with an existing file), and it's false by default. If think we need to check if the file is already present and then check if overwrite is False before we decide not to write down?